spec/sidekiq/form526_paranoid_success_polling_job_spec.rb

```diff
@@ -0,0 +1,141 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Form526ParanoidSuccessPollingJob, type: :job do
+  describe '#perform' do
+    let!(:new_submission) { create(:form526_submission) }
+    let!(:backup_submission) { create(:form526_submission, :backup_path) }
+    let!(:paranoid_submission_1) { create(:form526_submission, :backup_path, :paranoid_success) }
+    let!(:paranoid_submission_2) { create(:form526_submission, :backup_path, :paranoid_success) }
+    let!(:paranoid_submission_3) { create(:form526_submission, :backup_path, :paranoid_success) }
+    let!(:paranoid_submission_4) { create(:form526_submission, :backup_path, :paranoid_success) }
+    let!(:paranoid_submission_5) { create(:form526_submission, :backup_path, :paranoid_success) }
+    let!(:paranoid_submission_6) { create(:form526_submission, :backup_path, :paranoid_success) }
+    let!(:success_by_age) do
+      Timecop.freeze((1.year + 1.day).ago) do
+        create(:form526_submission, :backup_path, :paranoid_success)
+      end
+    end
+    let!(:accepted_backup_submission) do
+      create(:form526_submission, :backup_path, :backup_accepted)
+    end
+    let!(:rejected_backup_submission) do
+      create(:form526_submission, :backup_path, :backup_rejected)
+    end
+
+    context 'polling on parnoid success type submissions' do
```

Comment:
'paranoid' typo

Reply:
'paranoid success' is the correct term. I just didn't bother to type it out in the variable names for brevity [more info for posterity](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/disability/526ez/engineering_research/paranoid_success_submissions.md)

Reply:
JK i read `type` not `typo` because my brain makes typos as I read in real time

Reply:
fixed